### ISSUE TITLE
Missing reputation in Quest: Leaders of the Fang

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5032]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Currently when the quest is completed it awards the correct exp and item reward. However, the 350 Thunder Bluff reputation that should also be awarded upon completion is missing.
**Expected behaviour**: Tell us what should happen instead
There should be 350 Thunder Bluff reputation awarded.
**Steps to reproduce the problem**:

1. Start the quest chain till you reach this quest which is the final one in the chain
2. Collect the 4 gems from the leaders of the fang
3. Return to Nara Wildmane to complete the quest and check if any Thunder Bluff reputation is awarded upon completion

**Include proofs for this behaviour**
I forgot to take a screenshot but I have verified this issue by completing this quest with multiple Horde characters of different races. All produce the same results. I also cross-checked with http://wowwiki.wikia.com/wiki/Quest%3ALeaders_of_the_Fang to confirm that there should be a Thunder Bluff reputation gain.
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @wrathofthebug.
Issue confirmed on PTR.
Added suggested fix.
